CardsAgainst.xcodeproj/project.xcworkspace/xcshareddata/CardsAgainst.xcscmblueprint

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Could you please not check this in? See https://github.com/github/gitignore/pull/1547